### PR TITLE
[mds-db] schema null type additions

### PR DESCRIPTION
**Pull request by evanxadkins**
Non nullish enforcement additions to mds-db on event-processor related columns.

### CODE REVIEW

File: packages/mds-db/schema.ts [hunk 1]
```diff
@@ -337,25 +337,25 @@ const COLUMN_TYPE: { [C in COLUMN_NAME]: string } = {
...
-  [COLUMN.end_time]: 'bigint',
-  [COLUMN.event_counts]: 'json',
-  [COLUMN.event_time_violations]: 'json',
+  [COLUMN.end_time]: 'bigint NOT NULL',
+  [COLUMN.event_counts]: 'json NOT NULL',
+  [COLUMN.event_time_violations]: 'json NOT NULL',
   [COLUMN.event_type]: 'varchar(31)',
   [COLUMN.event_type_reason]: 'varchar(31)',
-  [COLUMN.events]: 'json[]',
+  [COLUMN.events]: 'json[] NOT NULL',
```
Comment (drtyh2o) [inline packages/mds-db/schema.ts]: json[] is interesting ... isn't an array of json just json?

File: packages/mds-provider-processor/package.json [hunk 2]
```diff
@@ -36,5 +36,7 @@
     "loglevel": "1.6.6",
     "uuid": "3.3.3"
   },
-  "devDependencies": {}
-}
\ No newline at end of file
+  "devDependencies": {
+    "husky": "^4.2.1"
+  }
```
Comment (drtyh2o) [inline packages/mds-provider-processor/package.json]: `devDependencies` should be in the root `package.json` ... `husky` is already installed.

File: packages/mds-provider-processor/package-lock.json [hunk 3]
```diff
@@ -0,0 +1,409 @@
+{
```
Comment (drtyh2o) [inline packages/mds-provider-processor/package-lock.json]: We don't use npm `package-lock` files ... please delete.

File: packages/mds-db/schema.ts (hunk 1 above)
Comment (evanxadkins) [reply packages/mds-db/schema.ts]: Is that true? I'm not too familiar with postgres data types but this has been working for me... I do want to investigate moving to jsonb types, they're binary forms that allow for more efficient storage and fetching so I'll play around with this one.

File: packages/mds-provider-processor/package.json (hunk 2 above)
Comment (evanxadkins) [reply packages/mds-provider-processor/package.json]: Not sure how this got there, will remove

**Comment by avatarneil** [comment]
@evanxadkins Do any of these require a migration?

**Comment by evanxadkins** [comment]
> @evanxadkins Do any of these require a migration?

My understanding is yes, but I want to hold off finalizing until Michael has a chance to run against his typeorm.

**Comment by avatarneil** [comment]
@evanxadkins is this PR necessary anymore?

**Comment by evanxadkins** [comment]
No not necessary anymore, I'll close.

**Pull request closed without merging**